### PR TITLE
add build instruction into README

### PR DESCRIPTION
**Pull request by gautric**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,39 @@ One could also use a standard Clang installation, build a sysroot from the
...
+sudo apt install ninja-build
+```
+
+## Build 
+
+To build the full package
+
+```shell script
+cd wasi-sdk
+NINJA_FLAGS=-v make package
```
[the remark this answers is not in the document]
Comment (gautric) [reply README.md:53]: do avoid any issue, I copied the github worflow cmd.
In this case, user builds with same prop as ci/cd

File: README.md [hunk 2]
```diff
@@ -22,6 +22,39 @@ One could also use a standard Clang installation, build a sysroot from the
...
+The Wasm-sdk's build process needs `ninja`.
+
+MacOS requirement
+```shell script
+brew install ninja
+```
+
+Linux (debian) requirement
+```shell script
+sudo apt install ninja-build
```
[the remark this answers is not in the document]
Comment (gautric) [reply README.md:44]: for sure for `cmake` and a fresh installation,
for `git` seriously ;-)

[the remark this answers is not in the document]
Comment (gautric) [reply README.md:44]: +1

**Comment by gautric** [comment]
@sbc100 I just changed with a list of package and a generic statement about installation

**Comment by sunfishcode** [comment]
This looks good to me!

**Pull request merged**